erizo/src/erizo/rtp/BandwidthEstimationHandler.cpp

```diff
@@ -143,6 +146,9 @@ void BandwidthEstimationHandler::updateExtensionMap(bool is_video, std::array<RT
 }
 
 void BandwidthEstimationHandler::read(Context *ctx, std::shared_ptr<DataPacket> packet) {
+  if (!initialized_) {
```

Comment:
should we forward the packet to the next handler in this case with `ctx->fireRead(std::move(packet));`?